apisix/cli/ngx_tpl.lua

```diff
@@ -766,6 +766,7 @@ http {
             grpc_set_header   "Host" $upstream_host;
             {% end %}
             grpc_set_header   Content-Type application/grpc;
+            grpc_set_header   Te trailers;
```

Comment:
TE instead of Te

Reply:
TE, te,Te is ok，it is not case-insensitve.  But I will use TE instead of Te in this commit.